source/common/res/features/import-notification/main.js

```diff
@@ -22,4 +23,3 @@
     setTimeout(poll, 250);
   }
 }());
-
```

Comment:
we should probably add an editorconfig -- lets keep the new line at end of files?